cmd/node_agent/na/gcp.go

```diff
@@ -53,3 +53,7 @@ func (na *gcpPlatformImpl) GetDialOptions(cfg *Config) ([]grpc.DialOption, error
 	options := []grpc.DialOption{grpc.WithPerRPCCredentials(&jwtAccess{jwtKey})}
 	return options, nil
 }
+
+func (na *gcpPlatformImpl) GetServiceIdentity(file string) (string, error) {
+	return "", nil
```

Comment:
How do we decide the service identity in GCE?

Reply:
It's undecided, in the jwt ideally.

Reply:
Leave a todo

Reply:
Then this shouldn't be a common method. In GCP we'll never find the identity from a file

Reply:
Chat offline, done.